### ISSUE TITLE
Fast User Switching issue

### ISSUE THREAD

**Issue by ledoktre** [tailscale/tailscale#10978]
### What is the issue?

I couldn't find where this is an existing issue on here, so posting it.

Trying to utilize multiple accounts on two separate self hosted servers but will not keep both available to fast user switch between. Additionally, when you hold option key and hit the menu bar icon, under the debug menu, we can add an account there and specify a different server, but it still tries to login to the default server.

### Steps to reproduce

Installed app from Mac App store.  

Used terminal to issue 'defaults write io.tailscale.ipn.macos ControlURL https://path.to.server' and signed in to first account.

Held option key and clicked on menu bar. Went to "Debug", and "Add Account".  It lets us specify another server here. I change the server, and click "Add Account" button. It opens a web browser, but on the original server.  

Can't get it to allow two accounts on two servers.

### OS

macOS

### OS version

13.6

### Tailscale version

1.58.2 (Mac App Store variant)

### Other software

None.

**Comment by ledoktre** [comment]
It appears like uninstalling the app store version and doing the standalone client (same 1.58.2 version number) works as expected.  Appears to be an issue isolated to just App Store Variant?

**Comment by catzkorn** [comment]
Hi,

Thanks for your report! We have flagged this with our macos team internally. By standalone, could you detail which [variant](https://tailscale.com/kb/1065/macos-variants) you mean?  

It sounds like you are unblocked for now.

**Comment by ledoktre** [comment]
Thanks for the reply. I got the standalone version here:

https://pkgs.tailscale.com/stable/#macos

It works as expected yes.  Just not as convenient for updating as App Store. App Store version is the one with the bug.

**Comment by agottardo** [comment]
@ledoktre we shipped a new account switcher today in Tailscale 1.60, with enhanced support for custom control servers. Did that improve your experience?

**Comment by ledoktre** [comment]
@agottardo I didnt see this message until just now. I was having some issues with the standalone pkg version on mac where it seems like the first time you connect there is no traffic, if you disconnect and reconnect it works fine. I thought maybe it was time to try the app store version again. 

I have version 1.66.3 installed right now, and try as I might, it will not switch servers.  Right now am trying to login to a new server but its not working.  If I use the instructions on /apple (on the headscale server) to hit the debug window, it does not launch anything.  If I go into Settings > Accounts and add an account it launches on whatever the default server is.

It would be nice to have this working (and ultimately figure out why I am having some connection issues on the first login on boot)

Thanks
